src/App.tsx

```diff
@@ -23,200 +174,139 @@ export const App: React.FC = () => {
-            4 items left
-          </span>
-
-          <nav className="filter" data-cy="Filter">
-            <a
-              data-cy="FilterLinkAll"
-              href="#/"
-              className="filter__link selected"
-            >
-              All
-            </a>
-
-            <a
-              data-cy="FilterLinkActive"
-              href="#/active"
-              className="filter__link"
-            >
-              Active
-            </a>
-            <a
-              data-cy="FilterLinkCompleted"
-              href="#/completed"
-              className="filter__link"
-            >
-              Completed
-            </a>
-          </nav>
+        {!!todoList.length && (
+          <>
 
-          <button
-            data-cy="ClearCompletedButton"
-            type="button"
-            className="todoapp__clear-completed"
-          >
-            Clear completed
-          </button>
-        </footer>
+            <footer className="todoapp__footer" data-cy="Footer">
+              <span className="todo-count" data-cy="todosCounter">
+                {`${counterActiveTodos} items left`}
+              </span>
+
+              <nav className="filter" data-cy="Filter">
+                <a
+                  data-cy="FilterLinkAll"
+                  href="#/"
+                  className={
+                    classNames('filter__link', {
+                      'filter__link selected': typeFilter === 'All',
+                    })
+                  }
+                  onClick={() => selectTypeFiltred('All')}
+                >
+                  All
+                </a>
+
+                <a
+                  data-cy="FilterLinkActive"
+                  href="#/active"
+                  className={
+                    classNames('filter__link', {
+                      'filter__link selected': typeFilter === 'Active',
+                    })
+                  }
+                  onClick={() => selectTypeFiltred('Active')}
+                >
+                  Active
+                </a>
+                <a
+                  data-cy="FilterLinkCompleted"
+                  href="#/completed"
+                  className={
+                    classNames('filter__link', {
+                      'filter__link selected': typeFilter === 'Completed',
+                    })
+                  }
+                  onClick={() => selectTypeFiltred('Completed')}
+                >
+                  Completed
+                </a>
+              </nav>
+
+              <button
+                data-cy="ClearCompletedButton"
+                type="button"
+                className={classNames(
+                  'todoapp__clear-completed hide', {
+                    hidden: !hasComplitedTodo,
+                  },
+                )}
+                onClick={clearCompletedTodo}
+              >
+                Clear completed
+              </button>
```

Comment:
I think would be better to create components here
![image](https://user-images.githubusercontent.com/110456763/200321575-b666d3c5-560f-4a4d-a96e-e37f204dfe3a.png)